go/chat/participantsource.go

```
@@ -165,7 +165,11 @@ func (s *CachingParticipantSource) GetNonblock(ctx context.Context, uid gregor1.
 func (s *CachingParticipantSource) GetWithNotifyNonblock(ctx context.Context, uid gregor1.UID,
 	convID chat1.ConversationID, dataSource types.InboxSourceDataSourceTyp) {
 	go func(ctx context.Context) {
-		s.sema.Acquire(ctx, 1)
+		err := s.sema.Acquire(ctx, 1)
```

Same here, just ignore the error